apps/src/code-studio/assets/SelectStartAnimations.jsx

```diff
@@ -121,6 +121,7 @@ export default class SelectStartAnimations extends React.Component {
         selectedAnimations={[]}
         hideCostumes={false}
         shouldWarnOnAnimationUpload={false}
+        uploadsEnabled={true}
```

Comment:
Why can we always set to true here?

Reply:
This component is used for content editors only. https://levelbuilder-studio.code.org/sprites